### ISSUE TITLE
How do I create Robolectric.jar file like "android-all-12.1-robolectric-8229987.jar" ?

### ISSUE THREAD

**Issue by simple-kss** [robolectric/robolectric#8079]
Hi. I am Android Developer and a big fan of Robolectric.
When I was testing the APK in Android U version, I observed that some Framework's API had been changed.
some methods was moved to other class in Framework class.

so, I want to create own "android-all-12.1-robolectric-8229987.jar" not "robolectric:4.6.1.jar".
(I had practiced making the robolectric:4.6.1.jar file refering ("https://robolectric.org/building-robolectric/"))

Because, I would like to change some methods. 
currently, in my development environment, in addition, I cannot access directly get a full platform robolectric jar like "android-all-12.1-robolectric-8229987.jar".

I need to make my jar file and I want to run test with it.

Could you help to let me know how do I create a full platform Robolectric.jar like "android-all-12.1-robolectric-8229987.jar"?

* I cannot use 'shadow' because new method is created in new Android version (U)

**Comment by utzcoz** [comment]
@simple-kss These jar files are built from AOSP, and maintained by Google's engineers like @hoisie and @brettchabot . @hoisie What about writting a wiki documentation to descibe how to build these jar files if other developers/users want to build it?

**Comment by hoisie** [comment]
https://github.com/robolectric/robolectric/wiki/Building-framework-JARs-from-platform-sources

TLDR is `m -j robolectric-host-android_all` :)

https://github.com/robolectric/robolectric/issues/6920#issuecomment-1072060246

**Comment by utzcoz** [comment]
@hoisie Sorry, I forget it.
